kernel/os/src/os_mempool.c

```diff
@@ -135,6 +135,45 @@ os_mempool_ext_init(struct os_mempool_ext *mpe, uint16_t blocks,
     return 0;
 }
 
+os_error_t
+os_mempool_clear(struct os_mempool *mp)
```

Comment:
Can this patch make use of this function already in os_mempool_init ?

Reply:
I was thinking on that too but in the end decided not to due to fact that clear takes initialized mempool